### ISSUE TITLE
Old .lz file and Last Updated date

### ISSUE THREAD

**Issue by Ante-Wessels** [parltrack/parltrack#60]
May be something goes wrong:

https://parltrack.org/dumps

has the new archive format .zst

Scripts downloading the older ep_meps.json.lz file still work. But it seems the .lz file is old, not updated. 

The page says Last Updated: 2024-11-11. But this seems not correct for the ep_meps.json.zst file, as it has newer information. It seems the update date is based on an old .lz file. 

All the Last Updated dates are somewhat old. 

Thanks!

**Comment by stef** [comment]
yes, we switched to zst format, due to the lz format being difficult to handle on apple and microsoft systems.

**Comment by Ante-Wessels** [comment]
I understand parltrack changed archive format. But the website seems only partially updated. 

For instance, the row with "Scraper Log" has as latest log "2024-11-11.log.zst (summary)". That is old and gives a 404. 

The previous dumps are: 
    2024-11-10 (summary)
    2024-11-09 (summary)
    2024-11-08 (summary)

They do work, but are old .lz files. The new log is available by rewriting the url. 

The field "Last Updated" reports "2024-11-11".

The whole column "Last Updated" reports "2024-11-11". That is not correct as the files are updated. 

Thanks!

**Comment by stef** [comment]
aaaah! now i get it! thx! will fix it asap.

**Comment by stef** [comment]
it was already fixed. i just forgot to restart the uwsgi server.
